ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -49,6 +50,10 @@ public class RadioTelescopeController
         // Snow dump timer
         private static System.Timers.Timer snowDumpTimer;
 
+        // Spectracyber timer
+        private static System.Timers.Timer spectraCyberTimer;
```

Comment:
We don't use the timer anymore so it can be removed

- [ ] Remove spectracyber timer